assignment4.Rmd

```diff
@@ -105,12 +111,41 @@ sweep(loadings, 2, colSums(loadings), "/") #sweep() computes each row as a propo
 
```

Comment:
We can display these proportions in a data frame, which we will use later to weigh variables.

D5 <- mutate_all(as.data.frame(loadings), funs((.)/sum(.)))